go.mod

```diff
@@ -5,40 +5,38 @@ go 1.24.1
 require (
 	github.com/dop251/goja v0.0.0-20240220182346-e401ed450204
 	github.com/go-logr/logr v1.4.2
-	github.com/google/cel-go v0.17.8
+	github.com/google/cel-go v0.20.1
 	github.com/kubestellar/kubeflex v0.9.0
 	github.com/onsi/ginkgo/v2 v2.22.0
 	github.com/onsi/gomega v1.36.1
-	github.com/prometheus/client_golang v1.18.0
+	github.com/prometheus/client_golang v1.19.1
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/time v0.7.0
 	google.golang.org/protobuf v1.35.1
-	k8s.io/api v0.30.12
-	k8s.io/apiextensions-apiserver v0.30.12
-	k8s.io/apimachinery v0.30.12
-	k8s.io/apiserver v0.30.12
-	k8s.io/cli-runtime v0.30.12
-	k8s.io/client-go v0.30.12
-	k8s.io/code-generator v0.30.12
-	k8s.io/component-base v0.30.12
+	k8s.io/api v0.31.13
+	k8s.io/apiextensions-apiserver v0.31.13
+	k8s.io/apimachinery v0.31.13
+	k8s.io/apiserver v0.31.13
+	k8s.io/cli-runtime v0.31.13
+	k8s.io/client-go v0.31.13
+	k8s.io/code-generator v0.31.13
+	k8s.io/component-base v0.31.13
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/kubernetes v1.30.12
-	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	open-cluster-management.io/api v0.15.0
-	sigs.k8s.io/controller-runtime v0.18.7
+	k8s.io/kubernetes v1.31.13
+	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
+	open-cluster-management.io/api v0.16.2
```

Comment:
I see that there is no release of ocm/api for Kube 1.31; we have to take this, which is for Kube 1.30, or the next, which is for Kube 1.32.

+1 on the choice made here.